package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/observablehq/stdlib.git"
   },
   "scripts": {
-    "test": "tap 'test/**/*-test.js'",
+    "test": "tap 'test/**/*-test.js' --reporter classic",
```

Comment:
What was the issue with the default reporter?

Reply:
exactly the same as https://github.com/tapjs/node-tap/issues/624

~~~bash
~/Sites/observablehq/stdlib fil/xlsx* 41s
❯ yarn test
yarn run v1.22.11
$ tap 'test/**/*-test.js'
Error: Cannot find module './reports/base'
Require stack:
- /Users/fil/Sites/observablehq/stdlib/node_modules/jackspeak/noop.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at resolveFileName (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/resolve-from/index.js:17:39)
    at resolveFrom (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/resolve-from/index.js:31:9)
    at module.exports (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/resolve-from/index.js:34:41)
    at importJSX (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/import-jsx/index.js:24:21)
    at module.exports (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/treport/lib/index.js:13:18)
    at exports.makeReporter (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:422:23)
    at runTests (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:744:3)
    at mainAsync (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:244:5)
    at main (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:127:3) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/fil/Sites/observablehq/stdlib/node_modules/jackspeak/noop.js'
  ]
}
TAP version 13
1..0
# time=0.973ms
----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |        0 |        0 |        0 |        0 |                   |
----------|----------|----------|----------|----------|-------------------|
error Command failed with exit code 1.
~~~

Reply:
Oh weird. I wasn't running into that.

Reply:
I just pushed a change to use the `base` reporter since it's the default, but with the explicit flag. let me know if that still works for you.

Reply:
No, unfortunately:

```
~/Sites/observablehq/stdlib visnup/xlsx*
❯ yarn test
yarn run v1.22.11
$ tap 'test/**/*-test.js' --reporter base
Error: Cannot find module './reports/base'
Require stack:
- /Users/fil/Sites/observablehq/stdlib/node_modules/jackspeak/noop.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at resolveFileName (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/resolve-from/index.js:17:39)
    at resolveFrom (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/resolve-from/index.js:31:9)
    at module.exports (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/resolve-from/index.js:34:41)
    at importJSX (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/import-jsx/index.js:24:21)
    at module.exports (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/node_modules/treport/lib/index.js:13:18)
    at exports.makeReporter (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:422:23)
    at runTests (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:744:3)
    at mainAsync (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:244:5)
    at main (/Users/fil/Sites/observablehq/stdlib/node_modules/tap/bin/run.js:127:3) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/fil/Sites/observablehq/stdlib/node_modules/jackspeak/noop.js'
  ]
}
```

Reply:
Found another reference to the issue: https://github.com/nodejs/citgm/issues/852#issuecomment-815378553

```
❯ node -v
v16.9.0
```